crates/astria-sequencer/src/app.rs

```diff
@@ -161,19 +158,20 @@ impl App {
             validator_address: None,
             executed_proposal_hash: Hash::default(),
             execution_result: HashMap::new(),
-            processed_txs: 0,
             current_proposer: None,
-            current_sequencer_block_builder: None,
+            write_batch: None,
+            app_hash: AppHash::default(),
```

Comment:
This is just a `Vec<u8>` <- let's toss the sha in here? It's already available at the crate root:

```rust
            app_hash: AppHash::try_from(crate::BUILD_INFO.git_sha.to_vec()).expect("TryFrom<Vec<u8>> for AppHash cannot fail"),
```

Reply:
why? this makes it non-deterministic which is bad, also it'll immediately get overwritten when `commit` is called after `init_chain`

Reply:
Think I misunderstood the purpose of app hash. :)